### PR TITLE
Fix overflow on org header

### PR DESCRIPTION
**Pull request by silverwind**
Fix overflow on org description because `.markup` was overwriting `overflow-wrap`.

image

### CODE REVIEW

File: templates/org/header.tmpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="ui container tw-flex">
 	{{ctx.AvatarUtils.Avatar .Org 100 "org-avatar"}}
-	<div id="org-info" class="tw-flex tw-flex-col">
+	<div id="org-info" class="tw-flex tw-flex-col tw-flex-1">
```
Comment (wxiaoguang) [inline templates/org/header.tmpl]: Maybe it also needs `tw-break-anywhere` in case the `org name` and others are too long

Comment (silverwind) [reply templates/org/header.tmpl]: Org name is limited to like 50 chars, but good to have, yes.

Comment (silverwind) [reply templates/org/header.tmpl]: Yeah works

image

Comment (silverwind) [inline templates/org/header.tmpl]:
```suggestion
	<div id="org-info" class="tw-flex tw-flex-col tw-flex-1 tw-break-anywhere">
```

**Review by wxiaoguang — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**